view/frontend/web/template/shipping.html

```diff
@@ -0,0 +1,93 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<li id="shipping" class="checkout-shipping-address" data-bind="fadeVisible: visible()">
```

Comment:
Might be good to have some `data-testid`s in here

Reply:
Added!